pkg/digger/digger.go

```diff
@@ -518,8 +518,7 @@ func cleanupTerraformOutput(nonEmptyOutput bool, planError error, stdout string,
 		endPos = strings.Index(stdout, matches[0]) + len(matches[0])
 	}
 
-	result = stdout[startPos:endPos]
-	return "```terraform\n" + result + "\n```"
```

Comment:
Not sure the impact of removing this "wrapper" formatting or if its still needed with the current change.

With it I would get 

![image](https://user-images.githubusercontent.com/17277004/234441791-ec150265-0d63-480c-9f61-471ff1d69a4b.png)